src/app/content/components/NudgeStudyTools/index.tsx

```diff
@@ -55,6 +57,17 @@ const NudgeStudyTools = () => {
 
   if (!positions) { return null; }
 
+  const ariaLabelKey = hasStudyGuides
+    ? 'i18n:nudge:study-tools:aria-label:with-study-guides'
+    : 'i18n:nudge:study-tools:aria-label:only-highlighting';
+
+  const messageKey = hasStudyGuides
+    ? 'i18n:nudge:study-tools:text:with-study-guides'
+    : 'i18n:nudge:study-tools:text:only-highlighting';
+
+  // tslint:disable-next-line: variable-name
+  const NudgeText = htmlMessage(messageKey, NudgeTextStyles);
```

Comment:
Creating components inside components changes their reference each render (at least in functional components) so they are always re-mounted, which probably won't cause any problems here but should generally be avoided.